carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/locator/internal/LocatingElementListHandler.java

```diff
@@ -95,8 +95,8 @@ public Object invoke(Object object, Method method, Object[] objects) throws Thro
 					 //do nothing and keep 'undefined' for control name 
 				}
 
-				ExtendedWebElement tempElement = new ExtendedWebElement(element, tempName);
-				tempElement.setBy(tempElement.generateByForList(by, i));
+				ExtendedWebElement tempElement = new ExtendedWebElement(element, tempName, by);
+//				tempElement.setBy(tempElement.generateByForList(by, i));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)